docs/Installing.asciidoc

```diff
@@ -217,20 +217,22 @@ The openQA web interface can be started via +MOJO_REVERSE_PROXY=1 morbo script/o
 development mode.
 
 +/var/lib/openqa/+ must be owned by root and contain several sub
-directories owned by the user that runs the workers (default 'geekotest'):
+directories owned by the user that runs openqa (default 'geekotest'):
 
-* +backlog+
 * +cache+ (cache directory, needs to be owned by the user who runs openqa)
 * +factory/iso+
 * +perl/+ (subdirectory 'autoinst' must link to +/usr/lib/os-autoinst+)
-* +pool/1+ (add more for each worker)
 * +share+ (shared directories for remote workers)
 * +testresults+
 * +tools+ (symlink to +/usr/share/openqa/tools/+)
+* ++db+ contains the sqlite database
 
-+/var/lib/openqa/db+ contains the sqlite database and must be owned by
-the user that runs the web interface ('geekotest'). To initially create this
-database, you need to run +tools/initdb+.
+To initially create the database, you need to run +script/initdb+.
+
+The worker needs to own +/var/lib/openqa/pool/$INSTANCE+, e.g.
```

Comment:
/var/lib/openqa/pool is owned by root in the package, so possibly we should discuss how agile we want our workers to be.